perfkitbenchmarker/linux_packages/maven.py

```diff
@@ -0,0 +1,76 @@
+import posixpath
```

Comment:
Can you add the copyright header?

@bvliu any idea why this wasn't auto flagged by https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/hooks/lib/check-boilerplate.sh

Reply:
@dlott done.